### ISSUE TITLE
[Accessibility] Feedback banner has insufficient contrast

### ISSUE THREAD

**Issue by elenm** [CityOfDetroit/bloom#220]
![image](https://user-images.githubusercontent.com/16543000/126221197-e876d344-80c9-4b6b-a2eb-89504d3776d2.png)
